static/styles/main.css

```diff
@@ -1,27 +1,33 @@
 .bloc {
     background-color: #e3f2fd;
     border-radius: 2px;
-    margin-bottom: 20px;
-    padding: 10px;
+    margin-bottom: 1em;
+    padding: 1em;
```

Comment:
I would use `rem` here. It prevents from weird side-effects in case of a CSS refactoring.